test/helpers/clear-element-test.html

```diff
@@ -13,18 +14,21 @@
         <div class="container"></div>
       </template>
     </test-fixture>
-    <script>
-      define(['clear-element'], (clearElement) => {
```

Comment:
Is there a reason for moving this inside the suiteSetup()? Not sure if it's clearer or not.